### DISCUSSION TITLE
Mastering Section Rendering APIs in Shopify Online Store 2.0 | Leo's dev blog – stories, insights, and ideas

### DISCUSSION THREAD

**Discussion by giscus** [hta218/leohuynh.dev#72] [Announcements]
# Mastering Section Rendering APIs in Shopify Online Store 2.0 | Leo's dev blog – stories, insights, and ideas

Learn how to leverage Section Rendering APIs to improve your Shopify store's performance and create dynamic, context-specific sections. Discover tips for querying dynamic section markup, including product and collection context

https://www.leohuynh.dev/blog/shopify-section-rendering-apis-notes

**Comment by hta218** [comment]
#### Hey, thanks for checking out this post! 💡

If you found it useful (or have some ideas for improvement), I’d love to hear from you! Leave a comment below with your thoughts, or give a reaction to show your support. It means a lot to me!

Cheers 🍺
